public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,85 @@
+# dvc.api.get_url()
+
+Return the URL to the storage location of the target data (`repo`/`path`).
```

Comment:
```
(`repo`/`path`)`
```

not sure I understand this notation? what does it mean

Reply:
If you expand the values of repo and path, it builds a full location. E.g. `gh.com/repo/path/to/data.txt` – of course it's not an actually valid URL like that but that's what this tries to convey.

I already use a similar parenthesis in `get` and `import` cmd ref descriptions: (`url`/`path`) so I thought to reuse it here.

I can say something like `path` in `repo` instead if this is confusing.

Reply:
Changing to ``(found at `path` in `url`)`` for `get` and `import`, and removing here to avoid confusion. Changing to "location of a data artifact tracked by DVC" and better explaining below. See d696063c